src/sources/formFields/SourceSystemFields.json

```diff
@@ -15,6 +15,25 @@
             "alias": "sourceSystemEndpoint",
             "type": "text"
         },
+        {
+            "name": "translatorType",
+            "alias": "sourceSystemTranslatorType",
+            "type": "select",
+            "options": [
```

Comment:
The options for the translatorType are a bit different: `dwca` en `biocase` . Those are the endpoint and then dwca supports dwc and biocase support abcd(efg). We don't really make a distinction between abcd and abcdefg.

Reply:
No, true, these were more a kind of placeholder as I didn't know what to do with these.
But as we discussed last week, it will be free text fields first instead of select from the data standard.
So in the next PR this will be changed :).